### PR TITLE
Add support for setting a System scope

### PR DESCRIPTION
**Pull request by ignatenkobrain**
Fixes: https://github.com/gophercloud/utils/issues/166
Signed-off-by: Igor Raits <igor.raits@gooddata.com>

### CODE REVIEW

File: openstack/clientconfig/requests.go [hunk 1]
```diff
@@ -656,6 +656,9 @@ func v3auth(cloud *Cloud, opts *ClientOpts) (*gophercloud.AuthOptions, error) {
 			} else if cloud.AuthInfo.DomainName != "" {
 				scope.DomainName = cloud.AuthInfo.DomainName
 			}
+			if cloud.AuthInfo.SystemScope != "" {
```
Comment (ZdenekPesek) [inline openstack/clientconfig/requests.go:659]: @ignatenkobrain 
would you mind to add `SystemScope` into section above to build that option also from ENV and have it all covered? :)
```
       if cloud.AuthInfo.SystemScope == "" {
               if v := env.Getenv(envPrefix + "SYSTEM_SCOPE"); v != "" {
                       cloud.AuthInfo.SystemScope = v
               }
       }
```

Comment (askarsabyrov) [reply openstack/clientconfig/requests.go:659]: @ignatenkobrain really need this fix, could you push proposed changes?

Comment (askarsabyrov) [reply openstack/clientconfig/requests.go:659]: @ZdenekPesek can I reopen this PR with those changes?

Comment (ignatenkobrain) [reply openstack/clientconfig/requests.go:659]: yes please, we sadly don't have much time/skills to push this through.

**Pull request closed without merging**